content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -192,13 +192,10 @@ A time zone database from the Go standard library is included in the binaries an
 ### Unsupported TimeZone specification
 
 Specifying a timezone using `CRON_TZ` or `TZ` variables inside `.spec.schedule`
-is **not officially supported** (and never has been).
-
-Starting with Kubernetes 1.29 if you try to set a schedule that includes `TZ` or `CRON_TZ`
-timezone specification, Kubernetes will fail to create the resource with a validation
-error.
-Updates to CronJobs already using `TZ` or `CRON_TZ` will continue to report a
-[warning](/blog/2020/09/03/warnings/) to the client.
+is **not officially supported** (and never has been). If you try to set a schedule
+that includes `TZ` or `CRON_TZ` timezone specification, Kubernetes will fail to
+create or update the resource with a validation error. Users are encouraged to use
```

Comment:
Style nit:
```suggestion
create or update the resource with a validation error. You should specify time zones using the
```

Reply:
Maybe we should mention that the early beta CronJob API didn't include that enforcement, so that people can understand how come they have `TZ` set in their CronJob.

Reply:
The idea is to completely drop the idea that it was allowed, that's why this strict statement.

Reply:
Also updated the style nit.

Reply:
> The idea is to completely drop the idea that it was allowed, that's why this strict statement.

Maybe we can word around it, but: there will be people who rely on a CronJob with a legacy-style timezone. Those people may well want to understand what's going on and how come their CronJob doesn't match the docs.

Reply:
That should trigger them to change their configs, to match the docs, so even though they are not official broken, they should be nudged to not use the unsupported bits.